luigi/interface.py

```diff
@@ -189,11 +205,11 @@ def run(*args, **kwargs):
 
     :param use_dynamic_argparse: Deprecated and ignored
     """
-    return _run(*args, **kwargs)['success']
+    return _detailed_summary_enabled(_run(*args, **kwargs), **kwargs)
```

Comment:
Let's not change this method. People should migrate to a another way of running if they want to run it proramtically.

Also this code looks super weird.

Reply:
> People should migrate to a another way of running if they want to run it proramtically.

Another way ? Instead of `luigi.run` ? 😢 That would be sad! 
made that a decorator and it looks much cleaner now! Please have a look @Tarrasch